### PR TITLE
Added API client creator and general entity creator helper methods

### PR DESCRIPTION
**Pull request by emulgeator**

### CODE REVIEW

**Comment by emulgeator** [comment]
Please have a look. Once this is done I can send the Storage tests for review too

File: src/Test/Integration/TDbContext.php [hunk 1]
```diff
@@ -15,6 +15,48 @@
      */
     abstract public function getDbConnection();
 
+    protected function createApiClient(string $token)
```
Comment (szeber) [inline src/Test/Integration/TDbContext.php]: Why is this here? It's quite specific to the storage service...
